packages/messaging/server/src/base/streamer.ts

```diff
@@ -78,21 +78,21 @@ export class Streamer {
   }
 
   public async send(url: string, data?: any, headers?: { [name: string]: string }) {
-    const config: AxiosRequestConfig<typeof data> = {}
-
-    if (headers) {
-      config.headers = headers
-    }
+    try {
+      const config: AxiosRequestConfig<typeof data> = {}
 
```

Comment:
Added after approval. Basically I moved the try so that it's over the entire function. Otherwise there is a small risk of unhandled rejection